src/linesearch/armijo_wolfe.jl

```diff
@@ -1,5 +1,40 @@
 export armijo_wolfe
 
+"""
+    t, good_grad, ht, nbk, nbW = armijo_wolfe(h, h₀, slope, g)
+
+Performs a line search from `x` along the direction `d` as defined by the
+`LineModel` ``h(t) = f(x + t d)``, where
```

Comment:
What do double backticks do?

Reply:
Math-like environment. Only evident in the docs.

Reply:
Ah ok, thank you.